pep-0591.rst

```diff
@@ -262,6 +262,17 @@ defined in the same module. Sealed classes seem most useful in
 combination with pattern matching, so it does not seem to justify the
 complexity in our case. This could be revisisted in the future.
 
+It would be possible to have the @final decorator on classes
+dynamically prevent subclassing at runtime. Nothing else in `typing`
```

Comment:
Single and double quotes make different meaning in .rst, I think here you need double quotes.